tooling/did-method-registry-entry.yml

```diff
@@ -41,6 +41,18 @@ properties:
     description: https://www.w3.org/TR/did-core/#dfn-verifiable-data-registry
     type: string
     maxLength: 512
+  implementation:
```

Comment:
we probably want to make this an array of URLs instead.... Seeing many entries here is a really good signal, especially if they are in different languages and by different organizations/people.

Reply:
I'd be fine w/ making this an array of URLs.

Reply:
+1 to making it an array.

Reply:
All of the above should probably be discussed somewhere in the docs ...

Reply:
-1 to defining a new "Informational Document"... but conceptually, I would be fine suggesting that the "DID Method Spec" which is already required, handle these concerns.... one less registry entry for us to maintain, and even more reason to take writing your method specification more seriously.

I think the rubric might be a better place to handle "implementations" since part of evaluating a did method is counting them.... and also, an evaluation of did:key implemented on Turing Machines will likely have much more complaints about performance than one built in RUST.